tests/test_ble_connection_discovery_client_targets.py

```
@@ -170,75 +184,83 @@ def test_connection_orchestrator_dispatch_set_event_and_kwarg_fallbacks(
+        orchestrator._thread_set_event("reconnected_event")
+
+        create_calls: list[dict[str, object]] = []
+
+        def _create_client(
+            _device: object, _cb: object, **kwargs: object
+        ) -> DummyClient:
+            create_calls.append(dict(kwargs))
+            if "pair_on_connect" in kwargs:
+                raise TypeError(
+                    "create_client() got an unexpected keyword argument 'pair_on_connect'"
+                )
```

Ruff (0.15.6)

[warning] Avoid specifying long messages outside the exception class

(TRY003)